client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -131,6 +132,7 @@ public HttpClient(HttpClientConfig config,
         collector = new ByteBufCollector(config.getMaxResponseSize());
         this.preRequestHooks = preRequestHooks;
         this.retryDuration = Duration.ofMillis(config.getRetryDelayMs());
+        setTimeout(config.getTimeoutMs(), ChronoUnit.MICROS);
```

Comment:
Shouldn't this be `ChronoUnit.MILLIS`?

Reply:
Effin' code-complete (MI -> MILLIS, MICROS or MILLENIA ;)